q2_feature_classifier/_skl.py

```diff
@@ -21,7 +21,8 @@
             'non_negative': True}],
           ['classify',
            {'__type__': 'custom.LowMemoryMultinomialNB',
-            'alpha': 0.01}]]]]
+            'alpha': 0.01,
+            'chunk_size': 20000}]]]]
```

Comment:
This parameter could be changed in `custom.py`, or here. I've changed it in both, for peace of mind.